handbook/ce/engaging-other-teams.md

```diff
@@ -1,5 +1,170 @@
 # Engaging Other Teams
 
-This is forthcoming (being worked on by [Virginia](https://about.sourcegraph.com/company/team#virginia-ulrich-she-her)) and is meant to consolidate/revise [CE-to-Engineering issue handover process](ce_to_eng_handover.md) and [How to route questions from customers](routing_questions.md). It will also touch on the fact that we never want CSEs to be gatekeepers and have a template for details CSEs will include when seeking help from engineering. And it will include details about engaging marketing, etc.
+Our goal as Customer Support is to only need help from engineering no more than 10% of the time (not counting when we file defects). It's okay that we need help, and we will need help, and we also continually take what we learn each time we do back to the rest of the team (and our product documentation, as relevant).
 
-Once drafted, Virginia will ask that all eng managers and CEs review
\ No newline at end of file
+When we need help, we seek it consistently, following the steps outlined here. We created these steps to ensure we:
+
+* Take the work as far as we can
+* Exhaust our collective expertise
```

Comment:
Points #2 and #3 seem to cover #1.  _As a team we aim to exhaust our collective expertise whilst minimizing time added to the customer experience whilst they wait_

This is just an observation, nothing to necessarily change :)